logger/logger_test.go

```diff
@@ -117,17 +117,17 @@ func validateError(log string, has bool, t *testing.T) {
 	}
 }
 
-// validateError validates for warning logs.
-func validateWarning(log string, has bool, t *testing.T) {
-	// 1. Validate for Warning function.
-	Warning("warning log")
-	if has != search(log, t, wPrefix, "warning log") {
-		t.Errorf("warning entry mishmash [result: %v]", !has)
-	}
-	// 2. Validate for Warningf function.
-	Warningf("warning log [%v,%s]", "param1", "param2")
-	if has != search(log, t, wPrefix, "warning log [param1,param2]") {
-		t.Errorf("warningf entry mishmash: [result: %v]", !has)
+// validateError validates for warn logs.
```

Comment:
` validateError` should be: `validateWarn`

Reply:
I'll fix it, the same mismatch is there for the other testing methods too.